common/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ set(pal_common_h_files
 
 set(pal_common_c_files
     src/gballoc.c
+    src/constbuffer.c
+    src/buffer.c

```

Comment:
Why are these part of PAL? They absolutely do not abstract anything that is HW or OS dependent....

Reply:
c_utils depends on PAL, therefore PAL cannot depend on c_utils and these things that it does depend on needed to go somewhere (that somewhere was decided to be here...for now).

---
In reply to: [450451168](https://github.com/Azure/azure-c-pal/pull/7#discussion_r450451168) [](ancestors = 450451168)